core/unit_test/incremental/Test16_ParallelScan.hpp

```diff
@@ -104,6 +104,164 @@ struct TestScan {
       ASSERT_EQ(h_data(i), upd);
     }
   }
+
+// Temporary: This condition will progressively be reduced when parallel_scan
+// with return value will be implemented for more backends.
+#if defined(KOKKOS_ENABLE_SERIAL) || defined(KOKKOS_ENABLE_OPENMP)
+#if !defined(KOKKOS_ENABLE_CUDA) && !defined(KOKKOS_ENABLE_HIP) &&             \
+    !defined(KOKKOS_ENABLE_OPENACC) && !defined(KOKKOS_ENABLE_SYCL) &&         \
+    !defined(KOKKOS_ENABLE_THREADS) && !defined(KOKKOS_ENABLE_OPENMPTARGET) && \
```

Comment:
This should also work for `Threads`, right?
```suggestion
    !defined(KOKKOS_ENABLE_OPENMPTARGET) && \
```

Reply:
No, I checked and for `Threads` there are additional changes required. There will be a separate PR for `Threads`